packages/asc-ui/src/components/TopTaskButton/TopTaskButtonStyle.ts

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import styled from '@datapunt/asc-core'
+import { themeSpacing, themeColor } from '../../utils'
+
+import IconStyle from '../Icon/IconStyle'
+
+export interface Props {
+  icon: React.ElementType
+  title?: string
+  content?: string
+  onDarkBackground?: boolean
+  width: string
+  className: string
+}
+
+const defaultProps = {
+  onDarkBackground: false,
+  width: '220px',
+}
+
+export default styled.div<Props>`
+  width: ${({ width = defaultProps.width }) => width};
+  padding: ${themeSpacing(2)} ${themeSpacing(4)};
+  cursor: pointer;
+  font-weight: bold;
+  font-size: 18px;
```

Comment:
We can remove the font-size / weight properties when we use the Paragraph component

Reply:
done